gremlin-javascript/src/main/javascript/gremlin-javascript/test/helper.js

```diff
@@ -27,8 +27,8 @@ const DriverRemoteConnection = require('../lib/driver/driver-remote-connection')
 const Client = require('../lib/driver/client');
 const PlainTextSaslAuthenticator = require('../lib/driver/auth/plain-text-sasl-authenticator');
 
-const serverUrl = 'ws://localhost:45940/gremlin';
-const serverAuthUrl = 'wss://localhost:45941/gremlin';
+const serverUrl = 'ws://gremlin-server-test-js:45940/gremlin';
```

Comment:
Can we add an environment variable based entry to this url? Like how go is set up. I'd imagine people running these tests individually outside of docker will encounter errors with this.